File: 181 excersises/161.cpp
#include <iostream>
#include <cmath>
using namespace std;

double factorialStirling(int n) {
    if (n == 0) return 1;
    return sqrt(2 * M_PI * n) * pow(n / M_E, n);
}

int main() {
    int n;
    cout << "Ingresa n: ";
    cin >> n;
    cout << "Aproximacion de " << n << "! ≈ " << factorialStirling(n) << endl;
    return 0;
}

File: 181 excersises/161.py
import math

def factorial_stirling(n):
    if n == 0: return 1
    return math.sqrt(2 * math.pi * n) * (n / math.e)**n

n = int(input("Ingrese n: "))
print(f"Aproximación de {n}! ≈ {factorial_stirling(n)}")